grammars/jsdoc.cson

```diff
@@ -252,28 +275,29 @@
         'name': 'variable.other.jsdoc'
       }
       {
-        'name': 'variable.other.jsdoc'
         'begin': '\\['
-        'end': '\\]|(?=\\*/)'
+        'beginCaptures':
+          '0':
+            'name': 'variable.other.jsdoc'
+        'contentName': 'variable.other.jsdoc'
```

Comment:
I don't like this.  I feel like it introduces complexities in the grammar, and I've noticed that while it's unnoticeable while _writing_ code, internally the scopes are still separated (such as in specs).

As you know I really value correct highlighting, but I think in this case it's better to use `name`.